packages/app/src/Element/Thread.tsx

```diff
@@ -254,15 +254,12 @@ const TierThree = ({ active, path, isLastSubthread, from, notes, related, chains
 };
 
 export interface ThreadProps {
-  this?: u256;
   notes?: TaggedRawEvent[];
 }
 
 export default function Thread(props: ThreadProps) {
   const notes = props.notes ?? [];
   const parsedNotes = notes.map(a => new NEvent(a));
-  // root note has no thread info
-  const root = useMemo(() => parsedNotes.find(a => a.Thread === null), [notes]);
```

Comment:
This doesn't work because there can be multiple root notes in the `parsedNotes` array.